openpype/hosts/nuke/plugins/publish/extract_review_data_mov.py

```diff
@@ -45,28 +45,65 @@ def process(self, instance):
         self.log.info(
             "StagingDir `{0}`...".format(instance.data["stagingDir"]))
 
+        self.log.info(self.outputs)
+
         # generate data
         with anlib.maintained_selection():
-            exporter = pnlib.ExporterReviewMov(
-                self, instance)
+            for o_name, o_data in self.outputs.items():
+                f_families = o_data["filter"]["families"]
+                f_task_types = o_data["filter"]["task_types"]
 
-            if "render.farm" in families:
-                instance.data["families"].remove("review")
-                data = exporter.generate_mov(farm=True)
+                test_families = any([
```

Comment:
This is super confusing for me. I've looked at the code for 5 minutes and I had to rewrite it to understand what are you trying to do...

Reply:
eh.. yeah you are right.. I will add comments to explain the process

Reply:
I hope it makes more sence now from comments